platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/WizAssetManager.java

```diff
@@ -177,8 +177,7 @@ public String getFile(String relpath) {
     public void deleteFile(String filePath) {
         try {
             // Delete file
-            String sqlsearch = "delete from assets where filePath= ?;";
-            database.rawQuery(sqlsearch, new String[] { filePath }).moveToFirst();;
+            database.delete(DATABASE_TABLE_NAME, "filePath like ?", new String[] { filePath + '%' });
```

Comment:
Ah, so you can delete all files in a particular folder in one go? :)
Why not create a `deleteFolder` for that instead? That would keep single file deletions faster, while at the same time you can ensure that your filePath ends with a slash.